src/os/bluestore/KernelDevice.cc

```diff
@@ -187,6 +189,72 @@ void KernelDevice::close()
   path.clear();
 }
 
+static string get_dev_property(const char *dev, const char *property)
+{
+  char val[1024] = {0};
+  get_block_device_string_property(dev, property, val, sizeof(val));
+  return val;
```

Comment:
Seems like it would  have done.  http://en.cppreference.com/w/cpp/language/implicit_conversion